### PR TITLE
Update MHST(U) code notes, added mine, reduced size

### PR DESCRIPTION
**Pull request by iSharingan**
slight reorder to Fort42 codes. Removed redundant/obsolete codes to reduce file size and prevent plugin crash when searching/adding codes. Added notes on PvP use for certain codes. Added character customization codes (flip gender of specific features/update changes to Rider Card).

### CODE REVIEW

File: db/00040000001BC500.txt [hunk 1]
```diff
@@ -621,31 +698,6 @@ D2000000 00000000
 
 [++PLAYER MODS++]
 
-[PLAYER LVL 99]
```
Comment (iSharingan) [inline db/00040000001BC500.txt]: redundant with above section's player stat codes

File: db/00040000001BC500.txt [hunk 2]
```diff
@@ -682,578 +734,7 @@ DD000000 00000180
 
 [Player Unlimited HP]
 1862D784 00000FFF
-
```
Comment (iSharingan) [inline db/00040000001BC500.txt]: obscenely verbose due to lack of offset jump pattern. Obsolete due to '999 all items after discard' code.

**Comment by JourneyOver** [comment]
@iSharingan We are not accepting pull requests for action replay codes, due to the fact that bernardo strictly wants to keep sharkive only to luma with gateshark and not nanquitas's action replay/CTRPF.. Yes there are a decent amount of codes in this repository that are steered toward the action replay/CTRPF direction at the moment but eventually they will be all culled and removed from this repository, it's just slow going.

So if you want this accepted you'll have to remove the notes and the [++ Name ++] [--] stuff.

For now I will keep this open until you either do the above, or come and close the issue yourself.

**Comment by iSharingan** [comment]
understood. I was trying to keep 'format' with the rest (and saw the request to submit codes here instead of on your database)

**Comment by JourneyOver** [comment]
@iSharingan you'll need to remove things like `{Softlocks pvp if you take enough damage for a normal KO}` as well. All notes like that have to be removed as lumas gateshark doesn't support notes.

You can still submit codes on my DB if you want, especially if they are Action replay oriented like you had before with the whole folder setup and stuff, it's just that setup isn't what bernardo wants for the codes on this DB. There will be an eventual split between my DB and this DB in that eventually all AR/CTRPF codes on this repo will be reworked/reworded for only gateshark, while on my own DB both AR/CTRPF and gateshark will be allowed.

**Comment by iSharingan** [comment]
apologies. imperfect round of whack-a-brace. fixed.
in that case I'll submit there as well.

File: db/00040000001BC500.txt [hunk 3]
```diff
@@ -654,25 +661,14 @@ DD000000 00000002
 D2000000 00000000
 {HOLD B WHILE RUNNING}
```
Comment (JourneyOver) [inline db/00040000001BC500.txt]: Missed one last note ;P

Comment (JourneyOver) [reply db/00040000001BC500.txt]: meant to mark that as requested changes, not approved .-.

**Comment by JourneyOver** [comment]
Alright there we go, everything looks good now! :)

**Pull request merged**